types/xmpp__client/browser.d.ts

```diff
@@ -0,0 +1 @@
+export * from './react-native';
```

Comment:
is this for this file?
https://github.com/xmppjs/xmpp.js/blob/main/packages/client/browser.js#L79-L81

Reply:
Yes. The interface is identical with the RN one.